internal/core/capi.cpp

```diff
@@ -1,3 +1,5 @@
+#include <functional>
```

Comment:
Why is the header needed even though nothing has change in the C++ source files?